oak_functions/wasm/src/tests.rs

```diff
@@ -446,20 +218,89 @@ fn create_test_wasm_state() -> WasmState<TestingLogger> {
 
 // Read the u32 value at the `address` from the Wasm memory.
 // Only needed in tests.
-fn read_u32(
-    ctx: &mut impl wasmi::AsContext,
-    memory: &mut wasmi::Memory,
-    address: AbiPointer,
-) -> Result<u32, OakStatus> {
-    let address = read_buffer(
-        ctx,
-        memory,
-        Src {
-            ptr: address,
-            len: 4,
-        },
-    )
-    .map_err(|_err| OakStatus::ErrInvalidArgs)?;
-    let address = LittleEndian::read_u32(&address);
-    Ok(address)
+fn test_read_u32(wasm_state: &mut WasmState<TestingLogger>, address: AbiPointer) -> u32 {
```

Comment:
I think that prefixing these with test feels a little confusing, since they are not tests, but helpers for the tests. They are already in a separate test file, so it is probably not needed to prefix them. But I don't have strong feelings about it, so if you think this is better, go ahead with it.

Reply:
Ha, yes, that is very true! I hadn't thought about this. What do you think about `mirrored_` as prefix---as they are essentially mirroring the functionality.

Reply:
I personally would not prefix it with anything. I think the fact that it is in a different file is enough to highlight the difference, plus you have the comments that clarify the intent.

If you really want to use different names, perhaps you can append something at the end to indicate that it has less error handling, but I don't think that is really needed either.